aliza/gulpfile.js

```diff
@@ -0,0 +1,21 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require ('gulp-mocha');
+
+gulp.task('default', ['lint', 'mocha'], () => {
+  console.log('default for lint and mocha');
+});
+
+
+gulp.task('lint', () => {
+  gulp.src('/*.js')
+    .pipe(eslint({}))
```

Comment:
I notice you didn't provide any options to lint, but your lint is passing on es6.  I'm curious how you did that, because I had to add a parse option to  allow ES6.